src/crypto.js

```diff
@@ -84,7 +84,6 @@ class Crypto {
     return resultText.join('\n');
   }
 
-
```

Comment:
Don't add all the changes in 1 commit, it makes reutrning to working version more complicated